.github/workflows/publish-to-test-pypi.yml

```diff
@@ -241,4 +243,15 @@ jobs:
         pytest -v -s --nbval-lax ./docs/notebooks/Llama_Stack_Building_AI_Applications.ipynb
         pytest -v -s --nbval-lax ./docs/notebooks/Llama_Stack_Benchmark_Evals.ipynb
 
-    # TODO: add trigger for integration test workflow & docker builds
+    - name: Integration tests
+      if: always()
+      run: |
+        pip install -U \
```

Comment:
Shall we directly do `llama stack build --template fireworks --image-type venv` to install these dependencies?